### PR TITLE
Viralflow dedup 1.2.0

### PR DESCRIPTION
**Pull request by costahud**
- fastp has been updated from version 0.20.1 to 0.23.4
- method dedup (deduplication) of fastp has been implemented
- new  repository for fastp 0.23.4 was uploaded to singularity library and configured
- parameter "-- dedup" (true or false) is availabe
- parameter "-- ndedup" (int from 1-6) is availabe

### CODE REVIEW

File: vfnext/main.nf [hunk 1]
```diff
@@ -42,7 +42,7 @@ ANSI_RESET = "\033[0m"
 
 log.info """
   ===========================================
-  VFNEXT v1.1.1
+  VFNEXT v1.0.1
```
Comment (dezordi) [inline vfnext/main.nf:45]:
```suggestion
  VFNEXT v1.2.0
```

File: vfnext/main.nf [hunk 2]
```diff
@@ -212,6 +214,7 @@ workflow {
   if (params.virus=="sars-cov2"){
     runPangolin(runVariantNaming_In_ch)
     runNextClade(runVariantNaming_In_ch, ref_fa)
+    // GAMBIARRA ALLERT --------------------------------------------------------
```
Comment (dezordi) [inline vfnext/main.nf:217]:
```suggestion
```

File: wrapper/viralflow [hunk 3]
```diff
@@ -77,6 +77,20 @@ parser.add_argument(
     help="organism reference genome code"
 )
 
+parser.add_argument(
+    "--dedup",
+    type=str,
```
Comment (dezordi) [inline wrapper/viralflow:82]:
```suggestion
    type=bool,
```

File: wrapper/viralflow [hunk 4]
```diff
@@ -77,6 +77,20 @@ parser.add_argument(
...
+    default=False,
+    help="enable dedup mode on fastp"
+)
+
+parser.add_argument(
+    "--ndedup",
+    type=str,
```
Comment (dezordi) [inline wrapper/viralflow:89]:
```suggestion
    type=int,
```

**Review by dezordi — CHANGES_REQUESTED** [review]

File: wrapper/viralflow [hunk 5]
```diff
@@ -77,6 +77,20 @@ parser.add_argument(
...
+    default=3,
+    help="standart accuracy level to dedup mode on fastp"
```
Comment (alexandrefreitasdasilva) [inline wrapper/viralflow:91]: help="standard accuracy level to dedup mode on fastp"

Comment (alexandrefreitasdasilva) [inline wrapper/viralflow:91]: `help="standard accuracy level to dedup mode on fastp"`

Comment (alexandrefreitasdasilva) [inline wrapper/viralflow:91]:
```suggestion
    help="standard accuracy level to dedup mode on fastp"
```

**Review by alexandrefreitasdasilva — COMMENTED** [review]
Adjust the version on warpper/viralflow

**Review by alexandrefreitasdasilva — APPROVED** [review]
All it is okay.

**Review by dezordi — APPROVED** [review]

**Pull request merged**